charts/csi-vxflexos/values.yaml

```diff
@@ -1,17 +1,23 @@
 ---
 # k8s/driver attributes
 
-# "version" is used to verify the values file matches driver version
```

Comment:
@adarsh-dell are you sure I should remove the version ?
Please see the comments from other values.yaml  : https://github.com/dell/helm-charts/blob/main/charts/csi-powermax/values.yaml#L50

Reply:
Good point @coulof .
@rishabhatdell Can you please give it a try and see the impact of this ?